osu.Game/Overlays/Profile/Header/Components/ProfileHeaderStatisticsButton.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osuTK;
+
+namespace osu.Game.Overlays.Profile.Header.Components
+{
+    public abstract class ProfileHeaderStatisticsButton : ProfileHeaderButton
+    {
+        private readonly OsuSpriteText drawableText;
+
+        protected ProfileHeaderStatisticsButton()
+        {
+            RelativeSizeAxes = Axes.Y;
```

Comment:
this doesn't make sense being sticked in here, instead of where it is created.

Reply:
It'd be less weird if this was doing `AutoSizeAxes = Axes.X` which I *think* was the intent here?

I don't even know what framework does if you turn off a relative-sized axis like this. I think it just hard-copies the current width in screen units, which means this may be wrong if this number ever changes (which I guess might not be an issue *right now*, but it's still a really strange thing to do).

Reply:
Now the height is defined in the base class. (https://github.com/ppy/osu/pull/11560/commits/4555b9ff704bad005b094d0e5997ed1b29e9606c)